README.md

```diff
@@ -46,9 +46,18 @@ application.  Executables for each dependency must be accessible through your
     * tools
 * iOS SDK (if building native iOS)
 
-#### Required Cordova Plugins
- from the root directory will install all dependencies.  The
-full list of requirements can be found by opening that script.
+### Required Cordova Plugins
+Below is a list of the required plugins. We have included a script that will fetch these automatically by simply running `python fetchPlugins.py`. This assumes that the locations of these repos are still as written below.
```

Comment:
The instructions for this are (at least I thought) included in the regular install steps.  It's probably worth juggling this section around so the doc isn't repetitive